mingw-w64-cmake/PKGBUILD

```
@@ -26,20 +26,20 @@ source=("https://www.cmake.org/files/v${pkgver%.*}/${_realname}-${pkgver}.tar.gz
         "qt5-static-plugin-support.patch"
         "pkg-config-without-prefix.patch"
         "do-not-generate-import-library-for-executables.patch")
-sha256sums=('d41462bdd80dc37f0d5608167b354bb3af8c068eee640be04c907154c5c113e2'
+sha256sums=('93d651a754bcf6f0124669646391dd5774c0fc4d407c384e3ae76ef9a60477e8'
             '76565ffc20010b27e895af7919f35deedc7fd0b86ca8746bbc22329185ad57fa'
             'aa816b734bf91941403b25bc9d9fd1797675ce2b4a7311bdb6e3ecae38964c37'
-            '5470eaf830aa43f37893d34af224915cc9be0c8b2d165a0115d7fd01881b9e69'
+            '3558259452883de78b176543de701d8a5757dbaf0d187b07e72256d97e13cabe'
             'aa807e20e4a2902a8e6f3f36e1cd7d4b043d0df12f4fb0f4e6bd27d9d6842c0f'
             '8503536bfc6606e147c62cd7bc28f984911a04b7bf8164264c345eacbf68dc5b')
 
 prepare() {
   cd ${_realname}-${pkgver}
-  patch -Np1 -i ${srcdir}/disable-response-files-for-msys.patch
-  patch -Np1 -i ${srcdir}/dont-install-bundle.patch
-  patch -Np1 -i ${srcdir}/qt5-static-plugin-support.patch
-  patch -Np1 -i ${srcdir}/pkg-config-without-prefix.patch
-  patch -Np1 -i ${srcdir}/do-not-generate-import-library-for-executables.patch
+  patch -Np1 -i "${srcdir}/disable-response-files-for-msys.patch"
+  patch -Np1 -i "${srcdir}/dont-install-bundle.patch"
+# patch -Np1 -i "${srcdir}/qt5-static-plugin-support.patch"
```

But you can't just silently drop a patch in that instance, you break a whole lot of things. If that happens, then you forgo the update to cmake 3.5.1 instead.